packages/thumbprint-tokens/src/__snapshots__/index.test.js.snap

```diff
@@ -362,3 +362,105 @@ $tp-wrap__no-pad-width: 1010px;
+    /// Maps to #2db783
+    public static let green: UIColor = UIColor(red: 0.1764706, green: 0.7176471, blue: 0.5137255, alpha: 1.0)
+    /// Maps to #109e78
+    public static let green500: UIColor = UIColor(red: 0.0627451, green: 0.61960787, blue: 0.47058824, alpha: 1.0)
+    /// Maps to #0f6648
+    public static let green600: UIColor = UIColor(red: 0.05882353, green: 0.4, blue: 0.28235295, alpha: 1.0)
+    /// Maps to #fdf7e7
+    public static let yellow100: UIColor = UIColor(red: 0.99215686, green: 0.96862745, blue: 0.90588236, alpha: 1.0)
+    /// Maps to #ffebb3
+    public static let yellow200: UIColor = UIColor(red: 1.0, green: 0.92156863, blue: 0.7019608, alpha: 1.0)
+    /// Maps to #ffdd80
+    public static let yellow300: UIColor = UIColor(red: 1.0, green: 0.8666667, blue: 0.5019608, alpha: 1.0)
+    /// Maps to #febe14
+    public static let yellow: UIColor = UIColor(red: 0.99607843, green: 0.74509805, blue: 0.078431375, alpha: 1.0)
+    /// Maps to #d48300
+    public static let yellow500: UIColor = UIColor(red: 0.83137256, green: 0.5137255, blue: 0.0, alpha: 1.0)
+    /// Maps to #8a5500
+    public static let yellow600: UIColor = UIColor(red: 0.5411765, green: 0.33333334, blue: 0.0, alpha: 1.0)
+    /// Maps to #ffeff0
+    public static let red100: UIColor = UIColor(red: 1.0, green: 0.9372549, blue: 0.9411765, alpha: 1.0)
+    /// Maps to #ffd9d9
+    public static let red200: UIColor = UIColor(red: 1.0, green: 0.8509804, blue: 0.8509804, alpha: 1.0)
+    /// Maps to #ffb0b0
+    public static let red300: UIColor = UIColor(red: 1.0, green: 0.6901961, blue: 0.6901961, alpha: 1.0)
+    /// Maps to #ff5a5f
+    public static let red: UIColor = UIColor(red: 1.0, green: 0.3529412, blue: 0.37254903, alpha: 1.0)
+    /// Maps to #cc3553
+    public static let red500: UIColor = UIColor(red: 0.8, green: 0.20784314, blue: 0.3254902, alpha: 1.0)
+    /// Maps to #76012c
+    public static let red600: UIColor = UIColor(red: 0.4627451, green: 0.003921569, blue: 0.17254902, alpha: 1.0)
+    /// Maps to #676d73
+    public static let black300: UIColor = UIColor(red: 0.40392157, green: 0.42745098, blue: 0.4509804, alpha: 1.0)
+    /// Maps to #2f3033
+    public static let black: UIColor = UIColor(red: 0.18431373, green: 0.1882353, blue: 0.2, alpha: 1.0)
+    /// Maps to #fafafa
+    public static let gray200: UIColor = UIColor(red: 0.98039216, green: 0.98039216, blue: 0.98039216, alpha: 1.0)
+    /// Maps to #e9eced
+    public static let gray300: UIColor = UIColor(red: 0.9137255, green: 0.9254902, blue: 0.92941177, alpha: 1.0)
+    /// Maps to #d3d4d5
+    public static let gray: UIColor = UIColor(red: 0.827451, green: 0.83137256, blue: 0.8352941, alpha: 1.0)
+    /// Maps to #ffffff
+    public static let white: UIColor = UIColor(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)
+}
+
+public enum Space {
+    public static let one: CGFloat = 4.0
```

Comment:
Can we add documentation comments to these as well? (e.g., `Spacer 1 – 4.0pt.` is what we're currently using in the iOS repo) Having the raw value in the documentation helps because it gets shown during autocomplete so you don't have to memorize all the name => raw value mappings when coding)